.github/workflows/build.yml

```diff
@@ -112,10 +109,9 @@ jobs:
         uses: actions/checkout@v3
 
       - name: Install stable toolchain
-        uses: actions-rs/toolchain@v1
+        uses: helix-editor/rust-toolchain@v1
         with:
           profile: minimal
-          toolchain: stable
```

Comment:
Why not run with both stable and the fixed version? That we can make sure helix code can still run on stable in case someone use it.

Reply:
Oh yeah we could have smoke tests on stable running on a cron or something. I just don't want it to fail CI for PRs when rust does a release.